### PR TITLE
Intake

### PR DESCRIPTION
**Pull request by cedricliu07**
- Added Intake Subsystem
- Added Intake Commands
- Mapped Commands to Buttons

### CODE REVIEW

**Review by edenreicherts — DISMISSED** [review]
I didn't see anything to change, great work!

**Comment by cedricliu07** [comment]
This needs to have the sim version added

File: src/main/java/frc/robot/subsystems/Intake/Intake.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems.Intake;
+
+import org.littletonrobotics.junction.Logger;
+
```
Comment (TheWonderfulWit) [inline src/main/java/frc/robot/subsystems/Intake/Intake.java:4]: white space

**Review by TheWonderfulWit — COMMENTED** [review]
white space

**Review by CanineSatsuma6 — APPROVED** [review]

**Review by edenreicherts — APPROVED** [review]

**Pull request merged**